data/presets/amenity/driver_training.json

```diff
@@ -0,0 +1,29 @@
+{
+    "icon": "maki-car",
+    "fields": [
+        "name",
+        "operator",
+        "operator/type",
+        "address",
+        "opening_hours"
+    ],
+    "moreFields": [
+        "license_classes",
```

Comment:
a field for [this tag](https://wiki.openstreetmap.org/wiki/Key:license_classes)  doesn't exist yet. you would need to create one in `data/fields/` for it.